jsapp/js/components/common/icon.tsx

```diff
@@ -36,12 +38,13 @@ class Icon extends React.Component<IconProps, {}> {
     const size = this.props.size || DefaultSize;
     classNames.push(`k-icon--size-${size}`);
 
+    const color = this.props.color || DefaultColor;
+    classNames.push(`k-icon--color-${color}`);
```

Comment:
Now I'm thinking it might be more elegant to add `k-icon--color-${color}` if `props.color` was provided, and add nothing if not provided. This way we don't really need the `inherit` default :)